### DISCUSSION TITLE
Image - Presentation type

### DISCUSSION THREAD

**Discussion by georgecm** [directus/directus#17490] [Feature Requests]
### Summary

In order to make it easier for editors to create things through Directus, I would like to be able to display some static (?) images on the edit/add view. The same way we now have Labels, Dividers and Buttons. 

### Basic Example

<img width="634" alt="image" src="https://user-images.githubusercontent.com/4643480/218253662-bd727054-0033-4faa-b973-63e3b9acb781.png">

### Motivation

I am building a page builder with components and I want to be able to remember what each component will look like. Ideally these could change based on a select button or tab, but at least they would be there to avoid me adding a button that sends people to an external image.

### Detailed Design

<img width="725" alt="image" src="https://user-images.githubusercontent.com/4643480/218253871-d7259057-ca2a-4268-a5c9-fc31ffb52829.png">

### Must Have:
- Display static images when editing an item

### Should Have:
- Some basic cropping function

### Could Have:
- Display an image/preview on the collection page
- The ability to change the image based on a Selector or Tab (which is another way to group that we miss)

### Drawbacks

I don't see any reason other than making the edit page a bit slower.

### Alternatives

Opening with a button but this is silly. People will need to go back and forth.

### Adoption Strategy

This should be straight forward as another option in the menu.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
